src/main/java/org/jabref/logic/citationkeypattern/BracketedPattern.java

```diff
@@ -846,51 +846,45 @@ static String allAuthors(AuthorList authorList) {
      */
     public static String authorsAlpha(AuthorList authorList) {
         StringBuilder alphaStyle = new StringBuilder();
-        int maxAuthors;
-        final boolean maxAuthorsExceeded;
-        if (authorList.getNumberOfAuthors() <= MAX_ALPHA_AUTHORS) {
-            maxAuthors = authorList.getNumberOfAuthors();
-            maxAuthorsExceeded = false;
-        } else {
-            maxAuthors = MAX_ALPHA_AUTHORS - 1;
-            maxAuthorsExceeded = true;
-        }
-
-        if (authorList.getNumberOfAuthors() == 1) {
-            String[] firstAuthor = authorList.getAuthor(0).getNamePrefixAndFamilyName()
-                                             .replaceAll("\\s+", " ").trim().split(" ");
-            // take first letter of any "prefixes" (e.g. van der Aalst -> vd)
-            for (int j = 0; j < (firstAuthor.length - 1); j++) {
-                alphaStyle.append(firstAuthor[j], 0, 1);
+        int numberOfAuthors = authorList.getNumberOfAuthors();
+        boolean andOthersPresent = numberOfAuthors > 1 &&
+                authorList.getAuthor(numberOfAuthors - 1).equals(Author.OTHERS);
+
+        if (numberOfAuthors == 1 || andOthersPresent) {
+            // Single author or "and others" case
+            String lastName = authorList.getAuthor(0).getFamilyName().orElse("");
+            if (lastName.startsWith("{") && lastName.endsWith("}")) {
+                alphaStyle.append(getOrganizationInitials(lastName));
```

Comment:
Rewrite using org.jabref.logic.formatter.bibtexfields.RemoveEnclosingBracesFormatter

1. format the string using that formatter
2. if result is not equal -> you found an organization
3. Pass that string to `getOrganizationInitials` - the first two lines can be removed then.

Reply:
Done.